File: Python/UberVan.py
from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []

    # La herencia de la familia car
    def __init__(self, license, driver, typeCarAccepted, seatsMaterial) -> None:
        super().__init__(license, driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial 

File: Java/UberVan.java
public class UberVan extends Car {
    
    String tytpecaraccepted;
    String seatsmaterial;
    public UberVan(String license, Account driver, String tytpecaraccepted, String seatsmaterial) {
        super(license, driver);
        this.tytpecaraccepted = tytpecaraccepted;
        this.seatsmaterial = seatsmaterial;
    }
}
